content/Community/contact.en.md

```diff
@@ -8,27 +8,37 @@ author: "Johan Ljunggren"
 
 #### #DSC channel
 
-The community gathers in the PowerShell #DSC channel which is an open
-forum for all DSC related questions, discussions and collaboration.
+The community gathers in the Virtual PowerShell User Group #DSC channel
+which is an open forum for all DSC related questions, discussions and
+collaboration.
 
 The PowerShell channels is bridged between Discord and Slack, so you
 can use either respectively app or website.
 
+There are not just the #DSC channel, there are a lot of other discussions
```

Comment:
I'd suggest:
`It's more than just the #DSC channel` to avoid the repetition of "There are"